modules/govuk/manifests/apps/content_performance_manager.pp

```diff
@@ -103,8 +103,16 @@
     app    => $app_name,
   }
 
-  govuk::procfile::worker {$app_name:
+  govuk::procfile::worker { 'content-performance-manager-google-analytics-worker':
```

Comment:
Can you not do `"${app_name}-google-analytics-worker"`?

Reply:
@andrewgarner done.